ArchiSteamFarm/Json/Steam.cs

```diff
@@ -390,9 +390,26 @@ internal class NumberResponse {
 
 		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
 		internal sealed class RedeemWalletResponse : EResultResponse {
+			internal sealed class InternalWallet {
+				[JsonProperty(PropertyName = "amount", Required = Required.Always)]
+				internal readonly ulong Amount;
+
+				[JsonProperty(PropertyName = "currencycode", Required = Required.Always)]
+				internal readonly ECurrencyCode KeyCurrencyCode;
```

Comment:
Wait, so **key** currency code is inside `wallet`? Did Valve really came up with crap like this?

Rename `InternalWallet` to `InternalKeyDetails`, rename `KeyCurrencyCode` to `CurrencyCode` (since now it's obvious). Rename `KeyRedeemingWallet` to `KeyDetails`, rename `AccountWalletCurrencyCode` to `WalletCurrencyCode` and finally we'll have it how Valve should make it to begin with.

God damn it who the hell alright with his mind put key details in `wallet` property, probably the same person who put wallet info in key validation.